EnglishFilter/sections/specific.txt

```diff
@@ -493,7 +493,7 @@ glosbe.com##div[id$="Banner"]
 hdmp4mania1.ga##p > a[href][target="_blank"] > img
 graphicux.com##.gux_smartbanner
 vexfile.com##.title-box > a[href^="https://tracker.vexfile.com/"] > img
-/api/spots/*$domain=yomovies.*|speedostream.com|watchomovies.com|gofilms4u.pro|pornleech.io|hentaihand.com|nhentai.com|filmlinks4u.pro
+/api/spots/*$domain=yomovies.*|speedostream.com|watchomovies.com|gofilms4u.pro|pornleech.io|hentaihand.com|nhentai.com|filmlinks4u.pro|myadultanimes.com
```

Comment:
These requests will be blocked by `||a.teenspirithentai.com^`, It's better to add this domain to rule which blocks third-party scripts

Reply:
I get popups without this rule.

Reply:
Which requests does this rule block?

Reply:
Don't need to complicate things. Just use this rule. 

![image](https://user-images.githubusercontent.com/76908410/114020463-32d6e480-98a2-11eb-927e-132e5696ba6f.png)

Reply:
> These requests will be blocked by ||a.teenspirithentai.com^

![Screenshot 2021-04-08 at 14 53 42](https://user-images.githubusercontent.com/18536328/114022396-630a8d00-987a-11eb-8f56-360866365502.png)

Reply:
I agree. But isn't it better to add to this rule? If the site changes adservers, this rule will be working perfectly.

Reply:
I agree. But isn't it better to add to this rule? If the site changes adservers, this rule will be working perfectly.

Reply:
@wout-codes 
some of third-party ad scripts don't have `/api/spots/` in their URLs, so it's better to add this domain to rule which blocks third-party scripts

Reply:
Ok, bootstrap regex.